.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -88,6 +88,55 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
 
+  build-wasm:
+    needs: [clang_format, license_check, rust_format_check, markdown_lint, shellcheck]
+    runs-on: ubuntu-22.04
+    env:
+      SCCACHE_CACHE_SIZE: 600M
+    steps:
```

Comment:
as a last step, could we check that the html file is created and optionally any other things that are stable paths like the .wasm bundle?

Use `test -f /path/to/file` to check if it's a file or `test -x /path/to/executable` to check if it's an executable file.

Assume as it's just a .html and .wasm you probably want like
```
     - name: Test output files exist
        run: |
          test -f ./build/../index.html
          test -f ./build/../bundle.wasm
```